lib/cisco_node_utils/bgp_neighbor_af.rb

```diff
@@ -110,10 +115,12 @@ def destroy
 
     # Returns ['<map1>', '<map2>']
     def advertise_map_exist
+      fail ArgumentError if platform == :ios_xr
```

Comment:
Why doesn't the default _exclude behavior (return nil for the getter, raise UnsupportedError for the setter) work for this property? Why do you need to explicitly raise an error?